### PR TITLE
Add new cask Firefly.app (firefly) v0.9.8

### PR DESCRIPTION
**Pull request by tddschn**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/firefly.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "firefly" do
+  version "0.9.8"
+  sha256 "f63c0d34913021f2dc4eb374abd02163613ccaf618497c76b5185a724035b136"
+
+  # raw.githubusercontent.com was verified as official when first introduced to the cask
+  url "https://raw.githubusercontent.com/cdtmirrors/yhc/master/yhc.dmg"
```
Comment (vitorgalvao) [inline Casks/firefly.rb:6]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/firefly.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask "firefly" do
+  version "0.9.8"
+  sha256 "f63c0d34913021f2dc4eb374abd02163613ccaf618497c76b5185a724035b136"
+
+  # raw.githubusercontent.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/firefly.rb:5]: `raw.githubusercontent.com/cdtmirrors/yhc`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by tddschn** [comment]
this app is no longer being maintained / updated but many people are still using it - is an `appcast` needed in this case?

File: Casks/firefly.rb (hunk 2 above)
Comment (tddschn) [reply Casks/firefly.rb:5]: this app is no longer being maintained / updated but many people are still using it - is an `appcast` needed in this case?

File: Casks/firefly.rb (hunk 1 above)
Comment (tddschn) [reply Casks/firefly.rb:6]: this app is no longer being maintained / updated but many people are still using it - is an `appcast` needed in this case?

**Comment by vitorgalvao** [comment]
Thank you, but we’d rather not accept apps that are already discontinued.

**Pull request closed without merging**